plugins/actix-web/Cargo.toml

```diff
@@ -13,19 +13,22 @@ repository = "https://github.com/paperclip-rs/paperclip"
 paperclip-core = { path = "../../core", version = "0.4.0", optional = true }
 paperclip-macros = { path = "../../macros", version = "0.5.0", features = ["actix"] }
 futures = "0.3"
-actix-service = "1.0"
+actix-service1 = { version = "1", package = "actix-service" }
+actix-service2 = { version = "2", package = "actix-service" }
 actix-web2 = { version = "2", default-features = false, optional = true, package = "actix-web" }
 actix-web3 = { version = "3", default-features = false, optional = true, package = "actix-web" }
+actix-web4 = { version = "4", default-features = false, optional = true, package = "actix-web" }
 parking_lot = ">=0.10, <0.13"
 serde_json = "1.0"
 once_cell = "1.4"
 openapiv3 = { version = "1.0.0-beta.5", optional = true }
 include_dir = { version = "0.7.2", optional = true }
 
 [features]
-actix = ["actix3"]
+actix = ["actix4"]
```

Comment:
Should we also remove "actix" for these other libs? Alternatively we could leave actix pointing to actix3 and then we wouldn't break compat?

Reply:
Yes, I just removed unused `actix` features. The place where it lasts are macros, but it is used internally so I think it's OK. I also updated `Makefile` and `actix-plugin.md` to use `actix4` feature.